app/graphql/types/test_result.rb

```diff
@@ -3,6 +3,8 @@
 module Types
   # Definition of the Profile type in GraphQL
   class TestResult < Types::BaseObject
+    enforce_rbac Rbac::REPORT_READ, Rbac::REPORT_EXPORT_READ
```

Comment:
This means that test results can be only accessed if the `REPORT_READ` and the `REPORT_EXPORT_READ` permissions are both set. I think that's not correct, so we either need to
* change the `authorized_to?` method to deal with `any?` instead of `all?` which would break the current check in the REST API (needs to be checked separately with AND)
* come up with something else